packages/clients/meldemichel/src/mapConfigurations.ts

```diff
@@ -16,8 +16,6 @@ const stadtplan = '453'
 const luftbilder = '452'
 export const hamburgBorder = '6074' // boundary layer for pins / geolocalization
 
-// TODO decide whether to use static hhborder.geojson (TODO update file - is there a difference since last time?)
```

Comment:
The `hhborder.geojson` is still present and should be deleted, if the todo is resolved

Reply:
The unused file has been deleted. The lib function `passesBoundaryCheck` takes care of the underlying problem.